docs/_articles/en/index.md

```diff
@@ -6,7 +6,7 @@ permalink: /
 
 The Salesforce Extension pack includes tools for developing on the Salesforce platform in the lightweight, extensible VS Code editor. These tools provide features for working with development orgs (scratch orgs, sandboxes, and DE orgs), Apex, Aura components, and Visualforce.
 
-![GIF showing Apex code completion, pushing source to a scratch org, and running Apex tests](../../images/overview.gif)
+![GIF showing Apex code completion, pushing source to a scratch org, and running Apex tests](./images/overview.gif)
 
```

Comment:
![Uploading Screen Shot 2022-03-22 at 11.58.02 AM.png…]()

Reply:
I don't think this image path is correct.